docs/api.md

```diff
@@ -1577,7 +1577,7 @@ In contrast to debounce, throttle simply drops events that occur more often than
 #### `stream.delay(delayTime) -> Stream`
 #### `most.delay(delayTime, stream) -> Stream`
 
-Timeshift a `stream` by `delayTime`.
+Timeshift a `stream` by `delayTime` in ms.
```

Comment:
Thanks @Sinewyk 👍 How about we spell it out fully: "... by `delayTime` in milliseconds"